product_default_image/models/res_company.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+
+    _inherit = 'res.company'
+
+    product_image_target = fields.Selection(
+        string='Default Product Image',
+        selection=[
+            ('global', 'Global'),
+            ('category', "Category's Image"),
```

Comment:
If category is chosen and the category doesn't have an image, does the global get used?